### ISSUE TITLE
Make drag-and-drop from palette keyboard-accessible

### ISSUE THREAD

**Issue by lmbateman** [camunda/camunda-modeler#5688]
### Problem you would like to solve

Keyboard accessibility for the drag-and-drop functionality in the palette has been requested. This will help us move toward satisfying [WCAG SC 2.5.7](https://www.w3.org/TR/WCAG22/#dragging-movements). I'm marking it as a feature request rather than a bug because it's a Level AA requirement.

### Proposed solution

I should be able to add elements to the canvas from the palette using only my keyboard.

There may be React libraries that can help with implementing this.

### Alternatives considered

Leave things as they are.

**Comment by nikku** [comment]
@lmbateman thanks for opening this issue. We can already model, keyboard only:

(1) every diagram starts with a start event
(2) you can append to any element via `a`
(2.1) however this starts to break (as expected) as soon as you want to do more complex work on the cyclic graph. BPMN is not "top to bottom", as standard web pages are.

Could you check how other open canvas tools (Google Maps, Miro, drawing tools) handle accessibility - if we have a good idea what a working E2E interaction is, then we can consider to invest. Otherwise I'd be inclined to not follow the "just make buttons accessible" route - this brings not enough value to our users. 

In general, I'd like to learn about your take about accessibility on the canvas.

**Comment by lmbateman** [comment]
> Could you check how other open canvas tools (Google Maps, Miro, drawing tools) handle accessibility - if we have a good idea what a working E2E interaction is, then we can consider to invest.

Some links (for me to read tomorrow 🙂 ):

- [Miro](https://medium.com/miro-engineering/miro-accessibility-introducing-keyboard-navigation-for-board-objects-dba41d0c4903)
- [Figma](https://help.figma.com/hc/en-us/articles/35063862380311-Accessibility-at-Figma)
- [Google Maps](https://support.google.com/maps/answer/6396990?hl=en&co=GENIE.Platform%3DDesktop)
- [Visio](https://support.microsoft.com/en-us/office/make-your-visio-diagram-accessible-to-people-with-disabilities-e2c847a9-f010-4fef-af65-16e252829d44#bkmk_testwin)
- [More Visio](https://support.microsoft.com/en-us/office/use-a-screen-reader-to-read-visio-diagrams-45ff89fc-fcd8-4c99-bc21-70eb9af29a61#OfficeVersion=Newer_versions)
- [PowerPoint](https://support.microsoft.com/en-us/office/use-keyboard-shortcuts-to-create-powerpoint-presentations-ebb3d20e-dcd4-444f-a38e-bb5c5ed180f4)

**Comment by barmac** [comment]
Moving to backlog as this requires dedicated effort.
